HicServices/RDMP
# Task/RDMP-65 remember last used extraction pipeline

Adds functionality to remember what pipeline was last used with an extraction.

To test:
* Create an extraction
* Select a pipeline
* Close the extraction tab
* reopen the extraction tab
* The last selected pipeline should be selected
